File: abc188/abc188_a/main.py
#!/usr/bin/env python3

import sys
sys.setrecursionlimit(10**7)

def main():
    X, Y = map(int, input().split())
    
    if X + 3 > Y and Y + 3 > X:
        print("Yes")
    else:
        print("No")


if __name__ == '__main__':
    main()

File: abc188/abc188_a/main.cpp
#include<iostream>
using namespace std;

int main(){
    int X, Y;
    cin >> X >> Y;

    if (X + 3 > Y && Y + 3 > X) {
        cout << "Yes" << endl;
    }
    else{
        cout << "No" << endl;
    }
    return 0;
}